psi4/driver/driver.py

```diff
@@ -2040,15 +2043,13 @@ def fchk(wfn, filename, debug=False):
     # Sometimes we have the SCF density on the reference_wavefunction and
     # othertimes we need to recompute it. Only affects correlated densities.
     if (module_ in ['DFMP2']):
-        print('False')
         fw.recompute_scf_density(False)
     else:
         fw.recompute_scf_density(True)
 
     if 'DCT' in fchk_label:
         # The Ca/Cb are the optimized orbitals from DCT. 
-        warn = " !! FCHK: In DCT calculations the SCF Density and DCT density entries are switched. !!"
-        core.print_out(warn)
+        core.print_out(" !! FCHKWriter: In DCT calculations the SCF Density and DCT density entries are switched. !!")
```

Comment:
Wouldn't it be better to switch these around in the fchk code?

Reply:
maybe? Not sure about what approach would be good. Shouldn't be too intrusive.
This patch is a short-term solution and once the underlying issues are solved, this whole thing needs to be re-written again.

Reply:
Added with latest commit a python-based fix for DCT. Not 100% sure this is a good idea. Else I would maybe turn of DCT.